example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-http/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
+            end
+          else
+            if ENV['REPO_NAME']
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/#{ENV['REPO_NAME']}"
+            end
+            if ENV['REPO_NAME'] == "on-tasks"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-tasks/" 
+            end
+            if ENV['REPO_NAME'] == "on-core"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-core/"  
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
+            end
+          end
+        end
+        
+        if ENV['CONFIG_DIR']
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/opt/monorail/"
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/opt/onrack/etc/"
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/home/vagrant/opt/monorail/"
+        end
 
+        if ENV['TEST_GROUP']		
+          if ENV['TEST_GROUP'] =~  /esxi-5-5-min-install.v2.0.test/ || ENV['TEST_GROUP'] =~  /esxi-5-5-max-install.v2.0.test/		
+            config.vm.provision "file", source: "./dhcpd.conf", destination: "~/dhcpd.conf"		
+            config.vm.provision "shell" do |s|		
+            s.inline = "cp /home/vagrant/dhcpd.conf /etc/dhcp"		
+            s.privileged = true		
+            end		
+          end		
+        end
+        if ENV['PYTHON_REPOS']
+          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
+          config.vm.provision "shell" do |s|		
+            s.inline = "apt-get install python-pip -y"		
+            s.privileged = true
+         end
+          list = ENV['PYTHON_REPOS'].split(' ')
+          list && list.each do |repo|
+            config.vm.provision "shell" do |s|		
+            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
+            s.privileged = true
+            end
+          end
+        end
+        
```

Comment:
agree to @DavidjohnBlodgett 

These steps is to volume host file to vagrant box for testing, in test the base vagrant box is v0.16, which is a box just for test.

With this no specific box version Vagrantfile, the latest released box will be pulled. Nothing need to do for a released box. RackHD services start automatically.